pyproject.toml

```
@@ -14,7 +14,7 @@ readme = "README.md"
 license = {text = "Apache License Version 2.0"}
 requires-python = ">=3.12"
 dependencies = [
-    "zigpy>=0.63.5",
+    "zigpy>=0.65.1",
```

If a new major zigpy version is released (0.66.0 instead of a minor bump), the suggestion below can be committed.
```suggestion
    "zigpy>=0.66.0",
```
(IMO, it should probably be a major bump, as an error will be logged. But the old API also stays working for now.)